packages/opentelemetry-api/src/metrics/BoundInstrument.ts

```diff
@@ -31,18 +28,8 @@ export interface BoundValueRecorder {
   /**
    * Records the given value to this value recorder.
    * @param value to record.
-   * @param correlationContext the correlationContext associated with the
-   *     values.
-   * @param spanContext the {@link SpanContext} that identifies the {@link Span}
-   *     which the values are associated with.
    */
   record(value: number): void;
-  record(value: number, correlationContext: CorrelationContext): void;
-  record(
-    value: number,
-    correlationContext: CorrelationContext,
-    spanContext: SpanContext
-  ): void;
```

Comment:
These signatures appeared completely unused. When get to the `BoundValueRecorder` they were just ignored, so I removed them. Have an open question in Gitter but figured I'd start with removing and can always add back if necessary.

Reply:
I believe these signatures came from the spec, but the spec on what to do with the correlations/span context was never finished.

Reply:
Ah, interesting. Feels like these should be left out until that is known but obviously I am new to the project and may not understand all the impacts.

I couldn't find any current usages. Obviously, may be some usages in the wild outside of these repos (js/contrib) but we are also < 1.0 so figured was OK to have a breaking change to clean-up at this point.